packages/core/src/model.js

```diff
@@ -1759,6 +1763,18 @@ ${associationOwner._getAssociationDebugList()}`);
     }).save(options);
   }
 
+  static _searchInclude(values) {
```

Comment:
New functions like this one should go in `model-internals.ts` instead

I think `getDefaultCreateInclude` may be a more explicit name